worlds/lingo/__init__.py

```diff
@@ -128,17 +128,21 @@ def create_items(self):
                     pool.append(self.create_item("Puzzle Skip"))
 
             if traps:
-                total_weight = sum(self.options.trap_weights.values())
+                trap_weights = self.options.trap_weights.value.copy()
+                if self.options.speed_boost_mode:
+                    trap_weights["Slowness Trap"] = 0
```

Comment:
It might be good to update the actual value in the option itself here, since it can be a bit confusing showing the old weight in the spoiler log